### PR TITLE
Main body branch

### PR DESCRIPTION
**Pull request by Olamarx**
I left the previous branch because it was messed.
I created the main tag and added the content to it.
I also continued the CSS I stated two days ago.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,5 +53,167 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
...
+          <ul class="pro-t">
+            <li class="pro-">html</li>
+            <li class="pro-">bootstrap</li>
+            <li class="pro-">Ruby</li>
+          </ul>
+             <button class="see-project" type="button">See Project</button>
+        </div>
+        <div class="pro-art-major">
+          <div class="art">
+           <h1 class="art-data">Profesional Art Printing Data</h1>
```
Comment (smunozmo) [inline index.html]: - [ ] You are almost there, however, as the previous reviewer said, please use a `hx` tag different from `h1`. There should be only one `h1` at the beginning of your project.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<!-- ![](https://img.shields.io/badge/Microverse-blueviolet) -->
-![This is an image](./images/check.jpg)
+
+![This is an image]()
```
Comment (smunozmo) [inline README.md]: - [ ] This image is still missing, I strongly suggest updating it, the README file is very important for people to know what your project is about.

File: index.html [hunk 3]
```diff
@@ -53,5 +53,167 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
...
+          <div class="about-me">
+            <div class="about-me-gen">
+              <h2 class="about-header">About me</h2>
+              <p class="hello-software">Hello I’m a software developer! I can help you build a product , feature or website Look through
+                some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+              </p>
+              <button class="button" type="button">Get My Resume</button>
+            </div>
+          </div>
```
Comment (smunozmo) [inline index.html:177]: - [ ] Good job, you are getting there, please fix the background problem with this section, here's the difference:

Template:
Captura de pantalla 2022-03-18 a la(s) 9 22 13 p  m

Your design:
Captura de pantalla 2022-03-18 a la(s) 9 21 57 p  m

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @Olamarx, 👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.

 ## Required Changes ♻️

- Please keep this in mind to avoid a review invalidation in the future:

⚠️  **Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise** ⚠️ 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Olamarx) [reply index.html]: @smunozmo are now saying I should return this to the previous branch?

File: README.md (hunk 2 above)
Comment (Olamarx) [reply README.md]: fixed!

File: index.html (hunk 3 above)
Comment (Olamarx) [reply index.html:177]: fixed

File: index.html [hunk 4]
```diff
@@ -53,5 +53,167 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
...
+            <li class="exp">html</li>
+            <li class="exp">bootstrap</li>
+            <li class="exp">Ruby</li>
+           </ul>
+           <button class="see-pro" type="button">See Project</button>
+          </div>
+        </div>
+        <div class="pro-art-major">
+          <div class="art">
+           <h3 class="art-data">Profesional Art Printing Data</h3>
```
Comment (MeqdamAlqudah) [inline index.html:81]: - [ Optional] Please make your page more responsive, make it reach 320 px screen width level 
image

File: index.html [hunk 5]
```diff
@@ -53,5 +53,167 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
...
+            <li class="pro-">bootstrap</li>
+            <li class="pro-">Ruby</li>
+          </ul>
+             <button class="see-project" type="button">See Project</button>
+        </div>
+        <div class="pro-art-major">
+          <div class="art">
+           <h4 class="art-data">Profesional Art Printing Data</h4> 
+            <p class="daily-s-con">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
```
Comment (MeqdamAlqudah) [inline index.html:97]: [OPTIONAl] Please use present tense to with your commit and make them describe an action example Add index.html , Fixing linter issues and so on 
image

**Review by MeqdamAlqudah — APPROVED** [review]
# Hi @Olamarx , 

### Status: Approved ✔️

:+1:  You have implemented all the suggested changes made by the previous reviewer. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Olamarx) [reply index.html:81]: fixed, thank you!

File: index.html (hunk 5 above)
Comment (Olamarx) [reply index.html:97]: Okay, noted, sir!

**Pull request merged**